### PR TITLE
[Iteration 2] v4.0-beta

### PR DESCRIPTION
**Pull request by Jeckerson**
Issue - #105

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+New BSD License
+
+Copyright (c) 2011-2018, Phalcon Framework Team
```
Comment (sergeyklay) [inline LICENSE.txt]:
```suggestion
Copyright (c) 2013-2019, Phalcon Framework Team
```

**Comment by Jeckerson** [comment]
Screen Shot 2019-08-24 at 00 53 44

File: app/library/Auth/Auth.php [hunk 2]
```diff
@@ -153,12 +163,10 @@ public function loginWithRememberMe()
 
         $user = Users::findFirstById($userId);
         if ($user) {
-
             $userAgent = $this->request->getUserAgent();
             $token = md5($user->email . $user->password . $userAgent);
```
Comment (ruudboon) [inline app/library/Auth/Auth.php]: Can we upgrade md5 to the bcrypt from the security component?

File: app/models/Users.php [hunk 3]
```diff
@@ -95,7 +104,7 @@ public function beforeValidationOnCreate()
         } else {
             $this->active = 'Y';
```
Comment (ruudboon) [inline app/models/Users.php]: Should we create a class constant for the Y & N?

File: app/library/Auth/Auth.php (hunk 2 above)
Comment (Jeckerson) [reply app/library/Auth/Auth.php]: Will do it in the next iteration, during mass tests creation, to make sure it works.

File: app/models/Users.php (hunk 3 above)
Comment (Jeckerson) [reply app/models/Users.php]: Thought about that, maybe will change here - #101

**Pull request merged**